backend/routers/database_setup.py

```
@@ -0,0 +1,19 @@
+# backend/routers/database_setup.py
+""" Database Setup Router - Initializes the database with necessary tables
+"""
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from .. import models, database
+
+router = APIRouter()
+
+@router.get("/init_db")
+def init_db(db: Session = Depends(database.get_db)):
```

Ruff (0.13.3)

Unused function argument: `db`

(ARG001)